javascripts/buzz.js

```diff
@@ -116,7 +116,11 @@ app.buzz = {
             }
             d.updatedDate = jQuery.timeago(new Date(d.sys_created));
             d.sys_description = d.sys_description[0].substr(0,197) + '...';
-            d.permanentLink = "//planet.jboss.org/post/" + d.sys_content_id;
+            if (d.sys_url_view.startsWith('http://developerblog.redhat.com')) {
```

Comment:
`sys_url_view` is an array, not a string:

![](http://wes.io/e6fu/content)

Change it to: `if(d.sys_url_view[0].startsWith(...)`